Application/Api/Billing/Bookkeeping/ApiBasketRepaymentAddPerson.php

```diff
@@ -0,0 +1,903 @@
+            $warning = new Warning('Bitte geben Sie mindestens 3 Zeichen in die Suche ein.', new Exclamation());
+        } elseif($Price <= 0) {
+
+            $warning = new Warning('Bitte geben Sie einen Betrag für die Gutschrift ein.', new Exclamation());
+        }
+        return $warning;
+    }
+
+    /**
+     * @param string        $Identifier
+     * @param string        $BasketId
+     * @param string        $DebtorSelection
+     * @param string        $Price
+     *
+     * @return Pipeline|string
+     */
+    public function addDebtorSelection($Identifier = '', $BasketId = '', $DebtorSelection = '', $Price = '')
+    {
+
+        $tblBasket = Basket::useService()->getBasketById($BasketId);
+        $tblDebtorSelection = Debtor::useService()->getDebtorSelectionById($DebtorSelection);
+        $PriceCorrection = round(str_replace(',', '.', $Price), 2);
+        if($PriceCorrection > 0){
+            $tblBasketVerification = Basket::useService()->createBasketVerification($tblBasket, $tblDebtorSelection, $PriceCorrection);
+            if($tblBasketVerification){
+                // schließt das Modal nicht
+                $Identifier = false;
+                return self::pipelineReloadTable($tblBasket->getId(), $Identifier);
+            } else {
+                return self::pipelineReloadTable($tblBasket->getId(), $Identifier);
+            }
+        }
+        return '';
+    }
+
+    /**
+     * @param string $BasketVerificationId
+     * @param string $Warning
+     *
+     * @return Well
+     */
+    public function showItemPrice($BasketVerificationId = '', $Warning = '')
+    {
+
+        $Causer = 'Beitragsverursacher nicht gefunden';
+        if(($tblBasketVerification = Basket::useService()->getBasketVerificationById($BasketVerificationId))){
+            if(!isset($_POST['Price'])){
+                $_POST['Price'] = $tblBasketVerification->getValue(true);
+            }
+            if(($tblPersonCauser = $tblBasketVerification->getServiceTblPersonCauser())){
+                $Causer = 'Beitragsverursacher: '.new Bold($tblPersonCauser->getLastFirstName());
+            }
+
+
+        }
+
+        $form = new Form(new FormGroup(new FormRow(array(
+            new FormColumn(
+                new Panel($Causer, array(new TextField('Price', '', 'Betrag der Gutschrift')), Panel::PANEL_TYPE_INFO)
+            ),
+            new FormColumn(
+                (new Primary('Speichern', self::getEndpoint(), new Save()))
+                    ->ajaxPipelineOnClick(self::pipelineCheckItemPrice($BasketVerificationId))
+            )
+        ))));
+        $form->disableSubmitAction();
+        if($Warning != ''){
+            $form->setError('Price', $Warning);
+        }
+
+        return new Well($form);
+    }
+
+    /**
+     * @param string $BasketVerificationId
+     * @param string $Price
+     *
+     * @return string
+     */
+    public function checkItemPrice($BasketVerificationId = '', $Price = '')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incorrect spacing between argument "$BasketVerificationId" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)